addons/website_slides/models/slide_question.py

```diff
@@ -16,9 +16,10 @@ class SlideQuestion(models.Model):
     slide_id = fields.Many2one('slide.slide', string="Content", required=True)
     answer_ids = fields.One2many('slide.answer', 'question_id', string="Answer")
     # statistics
-    attempts_count = fields.Integer(compute='_compute_statistics', groups='website_slides.group_website_slides_officer')
+    attempts_count = fields.Integer("Attempts", groups='website_slides.group_website_slides_officer', default=0)
     attempts_avg = fields.Float(compute="_compute_statistics", digits=(6, 2), groups='website_slides.group_website_slides_officer')
     done_count = fields.Integer(compute="_compute_statistics", groups='website_slides.group_website_slides_officer')
+    active = fields.Boolean(default=True, tracking=100)
```

Comment:
you could simply set tracking=1 as there is no other tracked field in this model